nucypher/blockchain/eth/actors.py

```diff
@@ -1275,7 +1279,7 @@ def perform_post_signature(
         )
         if async_tx:
             self.log.info(
-                f"Active signing ritual in progress: {self.transacting_power.account} has submitted tx "
+                f"Active signing ritual in progress: {self.threshold_signing_power.account} has submitted tx "
```

Comment:
The logging intention here is ambiguous: Do we want to log that an operator account submitted a TX or that a signer produced a signature? As far as I understand, the signer doesn't submit TXs, but operators do.